Added and updated Publishers

Saetron opened the pull request.

majora2007 commented:
These need to be compressed. You don't want anything over 5KB loading onto a user's page.

Saetron commented:
Images are compressed to under 5KB, added the file size limit to the about.txt and renamed it to a README.md. That way the information will show up in github when opening the publishers folder
![Screenshot 2024-09-19 191509](https://github.com/user-attachments/assets/7a8809ef-1d45-4353-85bf-1e1e7756319f)
